src/main/java/br/com/openenade/api/municipio/MunicipioRepository.java

```diff
@@ -0,0 +1,10 @@
+package br.com.openenade.api.municipio;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface MunicipioRepository extends JpaRepository<Municipio, Long>{
+
+    public void deleteByCodigo(Long codigo);
```

Comment:
There is no need to declare this method because JpaRepository already has delete by id method as default.